proto/greenfield/permission/common.proto

```diff
@@ -46,10 +46,11 @@ message Statement {
   // action_type define the operation type you can act. greenfield defines a set of permission
   // that you can specify in a permissionInfo. see ActionType enum for detail.
   repeated ActionType actions = 2;
-  // CAN ONLY USED IN bucket level. Support fuzzy match and limit to 5
-  // If no sub-resource is specified in a statement, then all objects in the bucket are accessible by the principal.
-  // However, if the sub-resource is defined as 'bucket/test_*,' in the statement, then only objects with a 'test_'
-  // prefix can be accessed by the principal.
+  // CAN ONLY USED IN bucket level. Support fuzzy match and limit to 5.
+  // The sub-resource name must comply with the standard specified in the greenfield/types/grn.go file for Greenfield resource names.
+  // If the sub-resources include 'grn:o:{bucket_name}/*' in the statement, it indicates that specific permissions is granted to all objects within the specified bucket.
+  // If the sub-resources include 'grn:o:{bucket_name}/test_*' in the statement, it indicates that specific permissions is granted to all objects with the `test_` prefix within the specified bucket.
+  // If the sub-resources is empty, it indicates that the permission will not be granted to any object within the specified bucket.
```

Comment:
the permission of objects will be granted for CREATE_OBJECT or LIST_OBJECT  even when the sub-resources is empty. So it may cause some misunderstandings here

Reply:
Yes, you must specify sub-resources, otherwise this permission policy is invalid.

Reply:
I went through the code again, and CreateObject doesn't check sub-resources. 

maybe for this comments, it should add `Excluding CreateObject`.

Reply:
yes,  and LIST_OBJECT may not need to check sub-resources too. It is normal to list objects even if the bucket has no objects.

Reply:
emm, I don't agree. Users may limit the scope of ListObject. For example, there is a bucket names `tests`, and two folders names `a/*`, `b/*`, and the owner only want other users list the objects with the common prefix `a`, not `b`.